s2/encode_fuzz_test.go

```diff
@@ -0,0 +1,32 @@
+// Fuzz tests for s2 decoding. Run using
+// go test -fuzz=Fuzz github.com/golang/geo/s2
+package s2
+
+import (
+	"bytes"
+	"testing"
+)
+
+func FuzzDecodeCellUnion(f *testing.F) {
+	cuCells := CellUnion([]CellID{
+		CellID(0x33),
```

Comment:
Do these cells matter at all? How were they generated?

Reply:
Nope, anything valid can be used as seed corpus. I copied this CellUnion from a test case. See https://go.dev/doc/security/fuzz/#requirements